templates/history.html

```diff
@@ -247,6 +247,39 @@ <h4 style="text-align: center;">Incident List is Currently Empty</h4>
                            {{ canarydrop['triggered_list'][item][field]['aws_keys_event_user_agent']|e }}
                          </td>
                         {% endif %}
+                        {% elif canarydrop['type'] ==  'wireguard' and field == 'src_data' %}
+                        <tr class="info_row">
+                        <td>
+                         Client Source Port
+                        </td>
+                        <td>
+                         {{ canarydrop['triggered_list'][item][field]['src_port'] |e }}
+                        </td>
+                       </tr>
+                       <tr class="info_row">
+                        <td>
+                         Client Handshake ID
+                        </td>
+                        <td>
+                         {{ canarydrop['triggered_list'][item][field]['session_index'] |e }}
+                        </td>
+                       </tr>
+                       <tr class="info_row">
+                        <td>
+                         Client Public Key
+                        </td>
+                        <td>
+                         {{ canarydrop['triggered_list'][item][field]['client_public_key'] |e }}
+                        </td>
+                       </tr>
+                       <tr class="info_row">
+                        <td>
+                         Server Public Key
+                        </td>
+                        <td>
+                         {{ canarydrop['triggered_list'][item][field]['server_public_key'] |e }}
+                        </td>
+                       </tr>
```

Comment:
there might need to be an `{%endif%}` here otherwise we may be doing all triggered_list only if wireguard type token